functional-tests/src/test/groovy/io/micronaut/gradle/fixtures/AbstractEagerConfiguringFunctionalTest.groovy

```diff
@@ -7,4 +7,12 @@ class AbstractEagerConfiguringFunctionalTest extends AbstractFunctionalTest {
     BuildResult build(String... args) {
         return super.build("tasks", *args)
     }
+
+
+    String getGraalVMBuilderExports() {
```

Comment:
Small thing but, can you not use `MicronautGraalPlugin.getGraalVMBuilderExports()` here?

Reply:
I can't because it would introduce a dependency that I don't want, for proper isolation (those tests make sure that everything works by _dynamically_ adding dependencies).